lib/srv/desktop/tdp/proto.go

```diff
@@ -463,13 +462,48 @@ func (m Error) Encode() ([]byte, error) {
 }
 
 func decodeError(in io.Reader) (Error, error) {
-	message, err := decodeString(in, tdpMaxErrorMessageLength)
+	message, err := decodeString(in, tdpMaxNotificationMessageLength)
 	if err != nil {
 		return Error{}, trace.Wrap(err)
 	}
 	return Error{Message: message}, nil
 }
 
+type Severity byte
+
+const (
+	SeverityInfo    Severity = 0
+	SeverityWarning Severity = 1
+	SeverityError   Severity = 2
+)
+
+type Notification struct {
```

Comment:
```suggestion
// Notification is an informational message sent from Teleport
// to the Web UI. It can be used for fatal errors or non-fatal
// warnings.
type Notification struct {
```

Add godocs